src/main/java/eu/europeana/metis/sandbox/config/IndexingConfig.java

```diff
@@ -102,6 +102,9 @@
   @Value("${sandbox.publish.solr.zookeeper.timeout:#{null}}")
   private Integer zookeeperPublishTimeoutInSecs;
 
+  @Value("${sandbox.application-name}")
```

Comment:
This is not at the right place, it should be part of the mongo configuration and in fact there should be one name for the preview indexer and one for the publish indexer.

Reply:
I see what you mean. I have fixed it